src/agent/agent.c

```diff
@@ -1010,6 +1018,81 @@ static const sd_bus_vtable internal_agent_vtable[] = {
         SD_BUS_VTABLE_END
 };
 
+/*************************************************************************
+ ********** org.containers.hirte.Agent.CreateProxy ***********************
+ ************************************************************************/
+static int agent_method_create_proxy(UNUSED sd_bus_message *m, void *userdata, UNUSED sd_bus_error *ret_error) {
+        Agent *agent = (Agent *) userdata;
+        if (agent == NULL) {
```

Comment:
This should never be able to happen. Not sure its really helpful to check it. If anything this should be an assert() to catch that something really weird is going on.

Reply:
Removed.